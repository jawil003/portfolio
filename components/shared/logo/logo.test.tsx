import { render } from "@testing-library/react";
import { Logo } from "./logo";

describe("Logo", () => {
  test("Create Logo", () => {
    const testLogo = render(<Logo />);

    expect(
      testLogo.baseElement,
    ).toMatchSnapshot();
  });
});
