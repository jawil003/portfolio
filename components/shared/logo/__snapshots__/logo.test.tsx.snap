// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Create Logo 1`] = `
.emotion-0 {
  padding: 20px;
  cursor: pointer;
  border-radius: 10px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  position: relative;
  overflow: visible;
}

.emotion-0>.logo-beta-label {
  background-color: #00739d;
  color: #f3f3f3;
  padding: 10px;
  font-size: 1.125rem;
  font-weight: 600;
  border-radius: 34px;
  position: absolute;
  bottom: 10px;
  right: -30px;
}

<body>
  <div>
    <div
      class="logo-container emotion-0"
    >
      <svg
        fill-rule="evenodd"
        height="60px"
        viewBox="0 0 1925 1191"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
          fill="url(#a)"
          transform="matrix(.43892 -.2534 .3315 .57417 -346.149 44.635)"
        />
        <path
          d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
          fill="url(#b)"
          transform="matrix(.43892 -.2534 .3315 .57417 334.96 34.958)"
        />
        <path
          d="M966.527 638.508c0-105.526-112.075-191.2-250.12-191.2-138.044 0-250.119 85.674-250.119 191.2V1928.35c0 105.52 112.075 191.2 250.119 191.2 138.045 0 250.12-85.68 250.12-191.2V638.508z"
          fill="url(#c)"
          transform="matrix(.43892 .2534 .3315 -.57417 767.637 1145.681)"
        />
        <defs>
          <lineargradient
            gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
            gradientUnits="userSpaceOnUse"
            id="a"
            x1="0"
            x2="1"
            y1="0"
            y2="0"
          >
            <stop
              offset="0"
              stop-color="#0078a3"
            />
            <stop
              offset=".27"
              stop-color="#00749e"
            />
            <stop
              offset=".5"
              stop-color="#00678d"
            />
            <stop
              offset="1"
              stop-color="#004a65"
            />
          </lineargradient>
          <lineargradient
            gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
            gradientUnits="userSpaceOnUse"
            id="b"
            x1="0"
            x2="1"
            y1="0"
            y2="0"
          >
            <stop
              offset="0"
              stop-color="#0078a3"
            />
            <stop
              offset=".27"
              stop-color="#00749e"
            />
            <stop
              offset=".5"
              stop-color="#00678d"
            />
            <stop
              offset="1"
              stop-color="#004a65"
            />
          </lineargradient>
          <lineargradient
            gradientTransform="matrix(8.55347 1718.32 -2247.83 6.53858 701.965 411.352)"
            gradientUnits="userSpaceOnUse"
            id="c"
            x1="0"
            x2="1"
            y1="0"
            y2="0"
          >
            <stop
              offset="0"
              stop-color="#0078a3"
            />
            <stop
              offset=".27"
              stop-color="#00749e"
            />
            <stop
              offset=".5"
              stop-color="#00678d"
            />
            <stop
              offset="1"
              stop-color="#004a65"
            />
          </lineargradient>
        </defs>
      </svg>
    </div>
  </div>
</body>
`;
